rel-eng/custom/custom.py

```diff
@@ -88,7 +88,7 @@ def filter_archives(path):
         rel_date = datetime.utcnow().strftime("%Y%m%d%H%M")
         self.release = rel_date + gitrev
         print("Building release: %s" % self.release)
-        run_command("sed -i '/^Release:/ s/\s\+/ 0.%s/' %s" % (self.release, self.spec_file))
+        run_command("sed -i '1i %%global nightly .%s/' %s" % (self.release, self.spec_file))
```

Comment:
my sed knowledge is rusty. `1i` means "insert at line 1" ?

Reply:
Exactly.

Reply:
right on.